app/views/catalog/_search_within_header.html.erb

```diff
@@ -2,7 +2,7 @@
 
   <wrapper class="search-results-heading" maxWidth="1125">
     <h1 class="top-content-title">
-      <%= t(render_search_to_page_header(params)) %>
+      <%= render_search_to_page_header(params) %>
```

Comment:
translation was moved into this helper I assume?

Reply:
We customized this and the customization was calling `t` when it shouldn't - I think this was probably broken before and relying on `t` falling back to whatever you passed to it if a key wasn't found.